scripts/find-lang.sh

```diff
@@ -13,6 +13,7 @@
 
 # 2011-11-16 Per Øyvind Karlsen <peroyvind@mandriva.org>
 #   * add support for HTML files (from Mandriva)
+#   * add support for multiple names
```

Comment:
This distinctly didn't happen in 2011.

Reply:
well, the changelog entry and the original commit (@cooker) did take place in 2011, but sure, the commit to rpm.org repo didn't..
Same for the previous ("add support for HTML files (from Mandriva)" that's already merged in master) too btw.
I'm occupied with other stuff currently, so don't expect any nitpicking commits from my end anytime soon, but please don't let me stop others from taking responsibility on meanwhile. ;)